src/plugins/note/note.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'jsx-dom';
+
+import {listen, memoize} from '@exadel/esl/modules/esl-utils/decorators';
+import {UIPPlugin} from '../../core/base/plugin';
+
+export class UIPNote extends UIPPlugin {
+  static is = 'uip-note';
+  static title = 'Note! ';
```

Comment:
```suggestion
  public static override is = 'uip-note';
  public static title = 'Note! ';
```

Reply:
Thanks! I'll udate it on the final preparations if not in that PR.